theme/scss/main.scss

```diff
@@ -6,14 +6,15 @@
 
 // Variables
 :root {
-  --app-bar-height: 75px;
+  --app-bar-height: 67px;
   --main-col-width: 775px;
   --side-col-width: 225px;
   --col-margin: 25px;
   --napari-columns: var(--side-col-width) var(--main-col-width);
 
-  @include breakpoint('sm') {
-    --col-margin: 50px;
+  @include breakpoint('xs') {
+    --app-bar-height: clamp(67px, 5vw, 75px);
+    --col-margin: clamp(25px, 11vw, 50px);
   }
```

Comment:
small question, since you're using clamps now, can this xs block be made the default at L9,12? otherwise LGTM! thanks for dealing with my nitpicking 😅

Reply:
No problem, I can be nitty sometimes too 😆 

I still needed to add a `@include breakpoint('sm')` with the maximum value for `--col-margin` because the clamp doesn't scale exactly between `300px` and `495px`. There's a small jump at `494px` and `495px`, but I think it's fine because I don't think people are going to be too focused on that when viewing the docs

Reply:
cool, could you add a comment in that section explaining that pls?

Reply:
I'll sneak a small commit to master with the comment added